Packs/Zerohack_XDR/README.md

```diff
@@ -0,0 +1,7 @@
+# xdr_integrations
+
+
```

Comment:
Please follow the best practices guide and templates for formatting and content. Use [this](https://github.com/demisto/content/blob/master/Packs/CortexXDR/README.md) as an example.